### ISSUE TITLE
Doesn't build with kernel 5.7.2

### ISSUE THREAD

**Issue by RoderickVM** [veeam/veeamsnap#7]
Dear maintainer:

This package doesn't build with kernels 5.7.2-zen1-1 and 5.7.2-arch1-1.

Please find the result message below:
```
==> dkms install veeamsnap/4.0.0.1961 -k 5.7.2-arch1-1
Error! Bad return status for module build on kernel: 5.7.2-arch1-1 (x86_64)

==> dkms install veeamsnap/4.0.0.1961 -k 5.7.2-zen1-1-zen
Error! Bad return status for module build on kernel: 5.7.2-zen1-1-zen (x86_64)
```
Additionally, attached is the /var/lib/dkms/veeamsnap/4.0.0.1961/build/make.log: [make.log](https://github.com/veeam/veeamsnap/files/4773665/make.log)

I appreaciate your help or pointing me in the right direction to sort this out, and get the veeam agent running again.

**Comment by k0ste** [comment]
Please see pinned post at AUR. Use this PKGBUILD and veeamsnap package will be builded flawless.
I think AUR maintainer should update the sources for AUR users, to avoid doubles of issues to this repo.

**Comment by k0ste** [comment]
Fixed in #5

**Comment by RoderickVM** [comment]
@k0ste Thank you very much. It worked flawlessly! Just had to update the patch file's SHA256SUM in the PKGBUILD, and run the installation again.
